WordPress/Classes/Extensions/Math.swift

```diff
@@ -22,14 +22,13 @@ extension Int {
 }
 
 extension Comparable {
-    /**
-     Clamps self between a minimum and maximum value
-     
-     - returns: the method returns
-        - min if self < min
-        - max if self > max
-        - otherwise it returns self
-     */
+    /// Clamps self between a minimum and maximum value
+    ///
+    /// - Returns: the method returns
```

Comment:
I actually started with lowercase keywords for `returns` / `parameter` / `parameters`, but ended picking up the capitalized version. Reason, as Diego said, it's human friendly. 😄